app/src/main/java/com/esri/arcgisruntime/opensourceapps/datacollection/views/fragments/PopupFragment.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.opensourceapps.datacollection.views.fragments
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.app.AlertDialog
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.activityViewModels
+import com.esri.arcgisruntime.opensourceapps.datacollection.R
+import com.esri.arcgisruntime.opensourceapps.datacollection.databinding.FragmentPopupBinding
+import com.esri.arcgisruntime.opensourceapps.datacollection.util.observeEvent
+import com.esri.arcgisruntime.opensourceapps.datacollection.viewmodels.IdentifyResultViewModel
+import com.esri.arcgisruntime.opensourceapps.datacollection.viewmodels.PopupViewModel
+import kotlinx.android.synthetic.main.fragment_popup.*
+
+/**
+ * Responsible for displaying PopupAttribute list.
+ */
+class PopupFragment : Fragment() {
+
+    private val popupViewModel: PopupViewModel by activityViewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val binding: FragmentPopupBinding = DataBindingUtil.inflate(
+            inflater,
+            R.layout.fragment_popup,
+            container,
+            false
+        )
+
+        val identifyResultViewModel: IdentifyResultViewModel by activityViewModels {
```

Comment:
This doesn't seem right. A PopupFragment shouldn't have a dependency to the IdentifyResultViewModel. It should only be aware of the PopupViewModel.

Reply:
Removed. I was able to fix the issue which was related to the app crashing when the user will tap on the bottomsheet.